src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -178,7 +179,16 @@ public World get(final WorldEvent e) {
 			public Block get(final StructureGrowEvent e) {
 				return e.getLocation().getBlock();
 			}
-		}, 0);
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(StructureGrowEvent.class, Block[].class, new Getter<Block[], StructureGrowEvent>() {
+			@Override
+			@Nullable
+			public Block[] get(final StructureGrowEvent e) {
```

Comment:
event instead of e

and no final in the event

Same for other additions modified. We have to deal with this as it comes up.